config/samples/heat_v1beta1_heat.yaml

```diff
@@ -3,27 +3,9 @@ kind: Heat
 metadata:
   name: heat
 spec:
-  customServiceConfig: "# add your customization here"
   databaseInstance: openstack
-  databaseUser: "heat"
   rabbitMqClusterName: rabbitmq
-  debug:
-    dbSync: false
-  heatAPI:
-    customServiceConfig: "# add your customization here"
-    debug:
-      service: false
-    resources: {}
-  heatCfnAPI:
-    customServiceConfig: "# add your customization here"
-    debug:
-      service: false
-    resources: {}
-  heatEngine:
-    customServiceConfig: "# add your customization here"
-    debug:
-      service: false
-    resources: {}
-  preserveJobs: false
+  heatAPI: {}
+  heatCfnAPI: {}
+  heatEngine: {}
```

Comment:
I have kind of been looking at the sample file as something that a user could reference to configure our operator. I can understand in the context of this change why you're doing this, but it is removing all references to things that can be configured.

I guess we need to discuss the intent behind this sample file and understand how we want it to be used. But potentially an alternative would be to copy this changed file into the tests/kuttl directory for use with these tests.

Reply:
while we can discuss how we maintain samples and test coverage, I'm wondering if we can unblock this PR to get a better test coverage. I can restore a few specific options if you think these are very useful, but I don't think all of the existing options such as databaseUser or serviceUser are very useful even of user references. Probably customServiceConfig ?